ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -199,6 +199,362 @@ def _grpc_channel_pool(self) -> GrpcChannelPool:
         return self._session_management_client._grpc_channel_pool
 
 
+class MultiplexerSessionHandler(_BaseSessionContainer):
+    """Manages multiplexer session information."""
+
+    def __init__(
+        self,
+        session_management_client: SessionManagementClient,
+        multiplexer_session_info: Optional[
+            Sequence[session_management_service_pb2.MultiplexerSessionInformation]
+        ],
+    ) -> None:
+        """Initialize multiplexer object."""
+        super().__init__(session_management_client)
+        self._multiplexer_session_cache: Dict[str, object] = {}
+        self._multiplexer_session_info: Sequence[MultiplexerSessionInformation] = []
```

Comment:
`Sequence[T]` is an immutable sequence, kind of like `IEnumerable<T>` in .NET. For a member variable, I would prefer using the actual type (which is `List[T]`). But see the next comment first.